applovin_max/ios/Classes/AppLovinMAXAdViewWidget.m

```diff
@@ -55,7 +55,7 @@ - (void)setCustomData:(NSString *)customData
     self.adView.customData = customData;
 }
 
-- (void)setAutoRefresh:(BOOL)autoRefresh
+- (void)setAutoRefreshEnabled:(BOOL)autoRefresh
```

Comment:
We should mark this property dynamic and implement the getter method too.

Reply:
Fair - @alhiwatan it will be good to understnd from ChatGPT what `@dynamic` is, we use it in our codebase all the time.

Last nit, in the header file we declare 

```
@property (nonatomic, copy, nullable) NSString *placement;
@property (nonatomic, copy, nullable) NSString *customData;
@property (nonatomic, copy, nullable) NSDictionary<NSString *, id> *extraParameters;
@property (nonatomic, copy, nullable) NSDictionary<NSString *, id> *localExtraParameters;
@property (nonatomic, assign, getter=isAutoRefreshEnabled) BOOL autoRefreshEnabled;
```

we should match the ordering in the implementation file as well. Like `- (void)setExtraParameters:(NSDictionary<NSString *, id> *)parameterDict` method should be declared right below `- (void)setCustomData:(NSString *)customData`.